ArduPlane/mode_qstabilize.cpp

```diff
@@ -15,29 +15,30 @@ bool ModeQStabilize::_enter()
 void ModeQStabilize::update()
 {
     // set nav_roll and nav_pitch using sticks
-    int16_t roll_limit = MIN(plane.roll_limit_cd, plane.quadplane.aparm.angle_max);
+    float roll_input = plane.channel_roll->norm_input();
```

Comment:
```suggestion
    const float roll_input = plane.channel_roll->norm_input();
```

Making it clear values are fixed can simplify thinking about the rest of the logic....

Reply:
done